R/teal.R

```diff
@@ -9,15 +9,8 @@
 #' @keywords internal
 "_PACKAGE"
 
-#' @importFrom magrittr %>%
-#' @export
-magrittr::`%>%`
-
-# Fix R CMD check notes
 #' @import shiny teal.data teal.slice
 #' @importFrom stats setNames
+#' @importFrom magrittr %>%
+#' @export %>%
```

Comment:
According to  [R Packages - Dependencies in Practice](https://r-pkgs.org/dependencies-in-practice.html#exports), the better way to re-export a method would be. And run devtools::check() to re-generate NAMESPACE and document.

```suggestion
#' @export
#' @importFrom magrittr %>%
magrittr::`%>%`
```

The @export directive requires fully qualified function names. ANd  it can not find documentation for %>% by placing them directly after the `@export` tag, as this would lead to CI errors. 

To correctly export the %>% operator, you must use the complete notation magrittr::\%>%`.